code/game/objects/items/hand_items.dm

```diff
@@ -151,6 +151,10 @@
 			if(istype(human_user.shoes, /obj/item/clothing/shoes/cowboy))
 				human_user.say(pick("Hot damn!", "Hoo-wee!", "Got-dang!"), spans = list(SPAN_YELL), forced=TRUE)
 				human_user.client?.give_award(/datum/award/achievement/misc/hot_damn, human_user)
+			else if(istype(human_user.w_uniform, /obj/item/clothing/under))
+				var/obj/item/clothing/under/uniform = human_user.w_uniform
+				if(istype(uniform.attached_accessory, /obj/item/clothing/accessory/lawyers_badge))
+					human_user.say("Objection!!", spans = list(SPAN_YELL), forced=TRUE)
```

Comment:
I think now that we have more than one thing that cares about this, it should just be made into a signal. It's going to be more work for you, but it's cleaner.

Reply:
I'll look into it